client/src/components/Navbar/Notifications.tsx

```diff
@@ -1,6 +1,6 @@
-import {  Badge, Divider } from '@mui/material';
+import { Avatar, Badge, Box, CircularProgress, Divider } from '@mui/material';
```

Comment:
btw we have styled LoadingIcon in LoadingIcon.tsx component for CircularProgress :)

Reply:
It is formatted differently and doesn't use for this particular application case. That is why I had to refactor it specifically for notifications usage